CMS/App_Data/CMSModules/Kentico.AcceleratedMobilePages/UI/Grids/AcceleratedMobilePages_AmpFilter/default.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <grid>
     <actions>
-      <action name="edit" caption="$General.Edit$" fonticonclass="icon-edit" fonticonstyle="allow" />
+      <action name="edit" caption="$General.Edit$" fonticonclass="icon-edit" fonticonstyle="allow" confirmation="$General.ConfirmEdit$" />
       <action name="#delete" caption="$General.Delete$" fonticonclass="icon-bin" fonticonstyle="critical" confirmation="$General.ConfirmDelete$" />
     </actions>
     <columns>
       <column source="PageNodeGuid" caption="AcceleratedMobilePages.GridPageName" externalsourcename="pagename"/>
       <column source="PageNodeGuid" caption="AcceleratedMobilePages.GridPath" externalsourcename="pagenamepath"/>
       <column source="StylesheetID" caption="AcceleratedMobilePages.GridCssStylesheet" externalsourcename="pagecss"/>
       <column source="UseDefaultStylesheet" caption="AcceleratedMobilePages.GridUseDefault" externalsourcename="usedefault"/>
-      <column source="AmpFilterLastModified" caption="AcceleratedMobilePages.GridLastModified"/>
+      <column source="AmpFilterLastModified" caption="AcceleratedMobilePages.GridLastModified" externalsourcename="usedefault"/>
```

Comment:
it doesn't make sense to map the last modified column to "usedefault" source name - https://github.com/Kentico/kentico-amp/blob/00ad8398bda860555e8864d8e0f5cc261ce3d6f9/Kentico.AcceleratedMobilePages/AmpFilterListExtender.cs#L78